integration_test/index_test.go

```diff
@@ -69,13 +70,13 @@ func TestKrewIndexList_NoIndexes(t *testing.T) {
 	defer cleanup()
 
 	test.WithEnv(constants.EnableMultiIndexSwitch, 1).WithIndex()
-	index := test.TempDir().Path("index/" + constants.DefaultIndexName)
+	index := test.TempDir().Path("index")
 	if err := os.RemoveAll(index); err != nil {
 		t.Fatalf("error removing default index: %v", err)
 	}
 
 	out := test.Krew("index", "list").RunOrFailOutput()
-	if !bytes.Equal(out, []byte("INDEX  URL\n")) {
+	if !strings.Contains(string(out), indexoperations.NoIndexMessage) {
 		t.Fatalf("expected index list to be empty:\n%s", string(out))
 	}
```

Comment:
testing with messages isn't good practice, let's not do that.
the previous test was good.

Reply:
Ok I'll change it back